frontend/.husky/pre-commit

```diff
@@ -1,5 +1,6 @@
 #!/bin/sh
 . "$(dirname "$0")/_/husky.sh"
 
-npx lint-staged
-cd frontend && npm run build
+cd frontend
+npm run lint-staged
+npm run build
```

Comment:
I know you only split this up but did not write it.
I have heard that people turn this hook off because building the front end on each commit takes too much time.
It would make more sense to get rid of the build part, only run formatting and lining and let the CI make sure the project can build.

Reply:
Nevermind, I took care of it with this PR.
https://github.com/ElektraInitiative/PermaplanT/issues/159